modules/statics/src/coins.ts

```diff
@@ -27,6 +27,12 @@ const XTZ_FEATURES = [...AccountCoin.DEFAULT_FEATURES, CoinFeature.ENTERPRISE_PA
 const CSPR_FEATURES = [...AccountCoin.DEFAULT_FEATURES, CoinFeature.REQUIRES_RESERVE];
 const ALGO_FEATURES = [...AccountCoin.DEFAULT_FEATURES, CoinFeature.SUPPORTS_TOKENS];
 
+export const batcherContractAddresses = {
```

Comment:
I'm not sure this is a great place for this, but putting it on the coin might be more trouble than it's worth. If you do want to do this though, here's what I'd do:

* Add a `batcherContractAddress?: string` property to the [`EthereumNetwork` interface](https://github.com/BitGo/BitGoJS/blob/db7d1c1819d31632c8d2a89387003f944443362a/modules/statics/src/networks.ts#L47)
* Add the `batcherContractAddress: '0x_the_address'` property to the implementation for Ethereum, Kovan, and Goerli ([here](https://github.com/BitGo/BitGoJS/blob/db7d1c1819d31632c8d2a89387003f944443362a/modules/statics/src/networks.ts#L218), [here](https://github.com/BitGo/BitGoJS/blob/db7d1c1819d31632c8d2a89387003f944443362a/modules/statics/src/networks.ts#L238), and [here](https://github.com/BitGo/BitGoJS/blob/db7d1c1819d31632c8d2a89387003f944443362a/modules/statics/src/networks.ts#L246))

Then you can get the correct batcher contract address for the current coin by capturing the 2nd param to `createInstance` (the statics coin, see [here](https://github.com/BitGo/BitGoJS/blob/5ad86846805b94eec3f125f33a8579286c3fc7d8/modules/core/src/v2/coinFactory.ts#L90). It's currently not being captured in the `createInstance` factory method). If you capture it as `this.staticsCoin`, then you can get the actual batcher address using `this.staticsCoin.network.batcherAddress`.

Reply:
Moved `batcherContractAddress` to the `EthereumNetwork` interface